### PR TITLE
[zh-cn] sync docs/reference/csi-node-v1

### PR DESCRIPTION
**Pull request by RifeWang**
…es/csi-node-v1.md

### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/csi-node-v1.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ api_metadata:
 content_type: "api_reference"
 description: "CSINode 包含节点上安装的所有 CSI 驱动有关的信息。"
 title: "CSINode"
-weight: 9
+weight: 4
+auto_generated: true
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/csi-node-v1.md:10]: In translated version, we intended to remove the `auto_generated` field.

Comment (RifeWang) [reply content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/csi-node-v1.md]: ok, removed

**Comment by RifeWang** [comment]
dup with https://github.com/kubernetes/website/pull/48209

**Pull request closed without merging**